mkdocs_meta_descriptions_plugin/plugin.py

```diff
@@ -22,7 +22,7 @@ def _get_first_paragraph_text(self, html):
         # Strip page subsections to improve performance
         html = re.split(self._headings_pattern, html, maxsplit=1)[0]
         # Select first paragraph directly under body
-        first_paragraph = BeautifulSoup(html, "html.parser").select_one("p")
+        first_paragraph = BeautifulSoup(html, "html.parser").select_one("p:not(div.admonition > p)")
```

Comment:
This skips the admonition `div` element if it exists and continues searching for the first paragraph.